C/Cling/build_tarballs.jl

```diff
@@ -0,0 +1,126 @@
+# Cling Homebrew Formula: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/cling.rb
+script = raw"""
+cd $WORKSPACE/srcdir
+mv llvm/ src
+mv clang/ src/tools/
+mv cling/ src/tools/
+
+LLVM_SRCDIR=$(pwd)/src
+
+# Patch to work-around missing clang runtime lib
+# (undefined symbol ___isPlatformVersionAtLeast).
+# See: https://github.com/JuliaPackaging/Yggdrasil/pull/5937#issuecomment-1334463175
+pushd ${LLVM_SRCDIR}
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/*
+popd
+
+cp src/LICENSE.TXT LICENSE_LLVM.TXT 
+cp src/tools/clang/LICENSE.TXT LICENSE_CLANG.TXT 
+cp src/tools/cling/LICENSE.TXT LICENSE_CLING.TXT 
+install_license LICENSE_LLVM.TXT 
+install_license LICENSE_CLANG.TXT 
+install_license LICENSE_CLING.TXT 
+
+# The very first thing we need to do is to build llvm-tblgen for x86_64-linux-muslc
+# This is because LLVM's cross-compile setup is kind of borked, so we just
+# build the tools natively ourselves, directly.  :/
+# See: https://github.com/JuliaPackaging/Yggdrasil/blob/86a4221f71ee8ca5c0fd1056829a1fe2b28c879b/M/Metal_LLVM_Tools/build_tarballs.jl#L35
+
+# Build llvm-tblgen and llvm-config
+mkdir ${WORKSPACE}/bootstrap
+pushd ${WORKSPACE}/bootstrap
+CMAKE_FLAGS=()
+CMAKE_FLAGS+=(-DLLVM_HOST_TRIPLE=${MACHTYPE})
+CMAKE_FLAGS+=(-DCMAKE_BUILD_TYPE=Release)
+CMAKE_FLAGS+=(-DLLVM_TARGETS_TO_BUILD="host;NVPTX")
+CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_HOST_TOOLCHAIN})
+CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING=OFF)
+cmake -GNinja ${LLVM_SRCDIR} ${CMAKE_FLAGS[@]}
+ninja -j${nproc} llvm-tblgen clang-tblgen llvm-config
+popd
+
+# Let's do the actual build within the `build` subdirectory
+mkdir ${WORKSPACE}/build
+cd ${WORKSPACE}/build
+
+CMAKE_FLAGS=()
+
+# Tell LLVM where our pre-built tblgen tools are
+CMAKE_FLAGS+=(-DLLVM_TABLEGEN=${WORKSPACE}/bootstrap/bin/llvm-tblgen)
+CMAKE_FLAGS+=(-DCLANG_TABLEGEN=${WORKSPACE}/bootstrap/bin/clang-tblgen)
+CMAKE_FLAGS+=(-DLLVM_CONFIG_PATH=${WORKSPACE}/bootstrap/bin/llvm-config)
+
+# Install things into $prefix
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
+
+# Explicitly use our cmake toolchain file and tell CMake we're cross-compiling
+CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN})
+CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING=ON)
+
+if [[ "${target}" == *apple* ]]; then
+    # Without this, cling will try to call `x86_64-apple-darwin14-clang++`
+    # on the macOS machine to find C++ header locations.
+    CMAKE_FLAGS+=(-DCLING_CXX_PATH=clang++)
+fi
+
+# Release build for best performance
+CMAKE_FLAGS+=(-DCMAKE_BUILD_TYPE=Release)
+
+cmake -GNinja ${LLVM_SRCDIR} ${CMAKE_FLAGS[@]}
+ninja -j${nproc} \
+    tools/cling/install \
+    install-clang-resource-headers
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macOS"),
+    Platform("aarch64", "linux"; libc = "glibc")
```

Comment:
Can maybe also try aarch64 darwin?
```suggestion
    Platform("aarch64", "linux"; libc = "glibc"),
    Platform("x86_64", "macOS"),
    Platform("aarch64", "macOS"),
```
?  And what about other platforms?